packages/components/src/font-size-picker/index.js

```diff
@@ -186,6 +186,7 @@ function FontSizePicker(
 								value={ option.value }
 								label={ option.label }
 								aria-label={ option.name }
+								tooltipText={ option.name }
```

Comment:
Should this be just a boolean, then reading from aria-label?

Reply:
I did have that in my first iteration, but changed it because a little voice inside my head asked: "_What if the aria label for the radio input button and tooltip text need to be different, for example, because we need to tell assisted technologies something else?"_

Also `aria-label` is optional on `ToggleGroupControlOption` so explicitly setting the `tooltipText` to determine both the display and content states at once seemed legit at the time.

To be honest, my reasoning wasn't more profound than that. 

Or is starting with Boolean and using `props[ 'aria-label' ] ?? label` for the text, and then and worrying about the other stuff later the safer approach?

Reply:
I'd echo Matías about the `boolean` and use `radioProps[ 'aria-label' ] ?? label`.

Reply:
Thanks for the confidence check, folks. I'll implement as suggested.